definition/latitude_longitude_properties.md

```diff
@@ -0,0 +1,58 @@
+## Various latitude/longitude properties
+
+
+### geom
+
+Find out more about the `geom` property [here](https://github.com/whosonfirst/whosonfirst-properties/tree/master/properties/geom).
+
+The `geom:latitude` and `geom:longitude` represent a geometric centroid for a record.
```

Comment:
`geometric or "math" centroid`

Reply:
Never mind, the next sentence addresses this and the machine-generated nature of the properties.

Reply:
it is the centroid used to generate intersections (the placetype) per @burritojustice 's Voronoi work

Reply:
(no fix here)